edit-post/assets/stylesheets/main.scss

```diff
@@ -219,18 +219,18 @@ body.gutenberg-editor-page {
 .editor-block-list__block {
 	input,
 	textarea {
-		// use opacity to work in various editor styles
+		// Use opacity to work in various editor styles.
 		&::-webkit-input-placeholder {
-			color: $dark-opacity-100;
+			color: $dark-opacity-300;
 		}
 		&::-moz-placeholder {
-			color: $dark-opacity-100;
+			color: $dark-opacity-300;
 		}
 		&:-ms-input-placeholder {
-			color: $dark-opacity-100;
+			color: $dark-opacity-300;
 		}
 		&:-moz-placeholder {
```

Comment:
Not sure why there's this version of the selector with one single colon. Shouldn't the one above with double colon be sufficient?

Reply:
Pretty sure that's copy pasta from https://stackoverflow.com/questions/19621306/css-placeholder-text-color-on-firefox where it's suggested that the single-colon version is necessary for Firefox 18 compatibility. Given we're on Firefox 61, and we have to support the last two versions, I think it's okay to drop that. Good catch.